### PR TITLE
update events links

### PR DESCRIPTION
**Pull request by Arhell**
resove https://github.com/opensourcedesign/opensourcedesign.github.io/issues/401
all links checked

### CODE REVIEW

**Review by AnXh3L0 — DISMISSED** [review]

File: _events/2016-03-18-fossasia.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ status: past
 permalink: /2016/03/18/fossasia-2016
 ---
 
-[FOSSASIA](http://2016.fossasia.org/) is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.
+[FOSSASIA](s) is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.
```
Comment (Erioldoesdesign) [inline _events/2016-03-18-fossasia.md:13]: Is this correct? looks like a link was replaced with an 's' to me??

Comment (Erioldoesdesign) [reply _events/2016-03-18-fossasia.md:13]: @Arhell

**Review by Erioldoesdesign — CHANGES_REQUESTED** [review]
Other than the FOSS Asia one I don't see any errors 😄

File: _events/2016-03-18-fossasia.md (hunk 1 above)
Comment (Arhell) [reply _events/2016-03-18-fossasia.md]: Done thanks

**Review by AnXh3L0 — APPROVED** [review]

**Review by SaptakS — APPROVED** [review]

**Comment by SaptakS** [comment]
@Erioldoesdesign this PR is waiting on your approval since you requested changes

**Review by Erioldoesdesign — APPROVED** [review]
LGTM

**Pull request merged**